pkg/certifier/certify/certify.go

```diff
@@ -70,7 +65,7 @@ func Certify(ctx context.Context, query certifier.QueryComponents, emitter certi
 		select {
 		case d := <-compChan:
 			if err := generateDocuments(ctx, d, emitter, handleErr); err != nil {
-				logger.Errorf("generate certifier documents error: %v", err)
+				return fmt.Errorf("generate certifier documents error: %w", err)
```

Comment:
Should this still be `logger`?

Reply:
Yes I can add the logger back in.

Reply:
Looking through this last night. The emitter error would be captured but this would be missed if it errors. So it has to return an error. I can also log the error at the same time.

Reply:
Oh, you're right. I see we're logging it on the caller site, so this LG.